core/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -237,6 +240,10 @@ private Path download(Terminal terminal, String pluginId, Path tmpDir) throws Ex
         }
 
         // fall back to plain old URL
+        if (pluginId.contains("://") == false) {
```

Comment:
Sorry, I meant those are not valid *URI*s. I can change the check to look for `":/"`